aiplatform/api/AIPlatform.Samples/PredictTextSummarizationSample.cs

```diff
@@ -0,0 +1,116 @@
+        string publisher = "google",
+        string model = "text-bison@001")
+    {
+        // Initialize client that will be used to send requests.
+        // This client only needs to be created once,
+        // and can be reused for multiple requests.
+        var client = new PredictionServiceClientBuilder
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var content = @"
+Background: There is evidence that there have been significant changes
+in Amazon rainforest vegetation over the last 21,000 years through the Last
+Glacial Maximum (LGM) and subsequent deglaciation. Analyses of sediment
+deposits from Amazon basin paleo lakes and from the Amazon Fan indicate that
+rainfall in the basin during the LGM was lower than for the present, and this
+was almost certainly associated with reduced moist tropical vegetation cover
+in the basin. There is debate, however, over how extensive this reduction
+was. Some scientists argue that the rainforest was reduced to small, isolated
+refugia separated by open forest and grassland; other scientists argue that
+the rainforest remained largely intact but extended less far to the north,
+south, and east than is seen today. This debate has proved difficult to
+resolve because the practical limitations of working in the rainforest mean
+that data sampling is biased away from the center of the Amazon basin, and
+both explanations are reasonably well supported by the available data.
+
+Q: What does LGM stands for?
+A: Last Glacial Maximum.
+
+Q: What did the analysis from the sediment deposits indicate?
+A: Rainfall in the basin during the LGM was lower than for the present.
+
+Q: What are some of scientists arguments?
+A: The rainforest was reduced to small, isolated refugia separated by open forest
+and grassland.
+
+Q: There have been major changes in Amazon rainforest vegetation over the last how
+many years?
+A: 21,000.
+
+Q: What caused changes in the Amazon rainforest vegetation?
+A: The Last Glacial Maximum (LGM) and subsequent deglaciation
+
+Q: What has been analyzed to compare Amazon rainfall in the past and present?
+A: Sediment deposits.
+
+Q: What has the lower rainfall in the Amazon during the LGM been attributed to?
+A:";
+
+        var instances = new List<wkt::Value>
+        {
+            wkt::Value.ForStruct(new()
+            {
+                Fields =
+                {
+                    ["content"] = wkt::Value.ForString(content),
+                }
+            })
+        };
+
+        var parameters = wkt::Value.ForStruct(new()
+        {
+            Fields =
+            {
+                { "temperature", new wkt::Value { NumberValue = 0 } },
+                { "maxDecodeSteps", new wkt::Value { NumberValue = 32 } },
```

Comment:
It's not clear to me which part of code in this sample summarizes the text rather that extracting sentiments from the text. I believe we should add comments to make this clear for the user. same goes for other similar samples as well.

Reply:
I agree it's not really summary but rather Q&A on the provided text. However, this is what they call it in Java, Node.js, Python samples, so I'm just following their naming.

Reply:
Is this explained in the docs where the sample is shown? It's really confusing.

Reply:
What actually changed apart from content ? Also what I'm trying to understand is that how can we say that this sample will summarize the provided text. Is it temperature or maxDecodeSteps value or model name or anything else ?

Reply:
Content is the main change (also some minor params change to match other languages). It summarizes because the content (prompt) tells the model to summarize.